inference-engine/samples/common/format_reader/CMakeLists.txt

```diff
@@ -40,3 +40,10 @@ set_target_properties(${TARGET_NAME} PROPERTIES COMPILE_PDB_NAME ${TARGET_NAME}
 if(COMMAND add_clang_format_target)
     add_clang_format_target(${TARGET_NAME}_clang FOR_TARGETS ${TARGET_NAME})
 endif()
+
+install(
+        TARGETS ${TARGET_NAME}
+        RUNTIME DESTINATION samples_bin/
```

Comment:
why `LIBRARY DESTINATION` is not used here as well? Here it's https://github.com/openvinotoolkit/openvino/blob/master/samples/CMakeLists.txt#L18-L19

The same for OpenCV C wrapper